src/pages/workspaces/workspace/analysis/modals/ComputeModal/AboutPersistentDiskSection.ts

```diff
@@ -0,0 +1,25 @@
+import { div, h, label } from 'react-hyperscript-helpers';
+import { Link } from 'src/components/common';
+import { handleLearnMoreAboutPersistentDisk } from 'src/pages/workspaces/workspace/analysis/modals/ComputeModal/persistent-disk-controls';
+import { computeStyles } from 'src/pages/workspaces/workspace/analysis/modals/modalStyles';
+
+export interface AboutPersistentDiskSectionProps {
+  setViewMode: () => void;
+}
+
+export const AboutPersistentDiskSection = (props) => {
+  const { setViewMode } = props;
+  return div({ style: { display: 'flex', flexDirection: 'column' } }, [
+    label({ style: computeStyles.label }, ['Persistent disk']),
+    div({ style: { marginTop: '0.5rem' } }, [
+      'Persistent disks store analysis data. ',
+      h(
+        Link,
+        {
+          onClick: () => handleLearnMoreAboutPersistentDisk({ setViewMode }),
```

Comment:
your other comment may have hinted at this... but the "direction" of this flow with setViewMode being drilled down into the sub-sub-component should be reversed.  Pushing setViewMode callback down the stack is locking in semantic "knowledge" of outer component mechanics to the inner inner component.

I recommend reversing this to instead be an onLearnMore event handler that the inner component exposes, and the outer components bubble up/out.  Then the component/s that house the setViewMode state mechanic can leverage that knowledge.  If you want the inner component to always handle the event tracking, that's fine.  Have it call the provided onLearnMore just before that.

Reply:
Refresh the page ;)